<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditStage</class>
 <widget class="QDialog" name="EditStage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>359</width>
    <height>291</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Stage</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Stage name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Distance (km)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameInput">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinInput">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Stages List</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closeStage">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>250</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="removeStage">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>120</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove selected</string>
   </property>
  </widget>
  <widget class="QListWidget" name="stageList">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>191</width>
     <height>101</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::MultiSelection</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="addStage">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>20</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add stage</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
